protobuf/lib/src/protobuf/field_set.dart

```diff
@@ -74,12 +73,18 @@ class _FieldSet {
   /// code as an `int`.
   Object _frozenState = false;
 
+  /// The [BuilderInfo] for the [GeneratedMessage] this [_FieldSet] belongs to.
+  ///
+  /// WARNING: Avoid calling this for any performance critical code, instead
```

Comment:
I see that `meta` is now plumbed through to several methods in accordance with this comment. Just curious: what exactly is the source of the slowdown? It's not immediately obvious to me why accessing the same reference one way would be less efficient than the other.

Reply:
I have *zero* idea. CC @sigurdm